website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -0,0 +1,379 @@
+---
+title: "Python models"
+---
+
+:::info Beta
+
+This feature is arriving in dbt Core v1.3, which is currently in beta. Note that only [specific data warehouses](#specific-data-warehouses) support dbt Python models.
+
+We encourage you to:
+- Read [the discussion](https://github.com/dbt-labs/dbt-core/discussions/5261)
+- Join the **#beta-feedback-python-models** channel in the [dbt Community Slack](https://www.getdbt.com/community/join-the-community/)
+
+Below, you'll see sections entitled "Our questions." We're working to develop our opinionated recommendations ahead of the final release this October—and you can help!
+:::
+
+## What is a Python model in dbt?
+
+A dbt Python model is a function that reads in dbt sources or models, applies a series of transformations, and returns a transformed dataset. Just like SQL models!
+
+Unlike SQL models, which return a `select` statement, each Python model returns a single <Term id="dataframe">data frame</Term>. When you run a Python model, the result of that data frame will be saved as a table in your data warehouse.
+
+Each Python model lives in a `.py` file in your `models/` folder. It defines a function named **`model()`**, which has two parameters:
+- **`dbt`**: A class compiled by dbt Core, unique to each model, that enables you to run your Python code in the context of your dbt project and DAG.
+- **`session`**: A class representing the connection to the backing engine, which allows you to interact with the data platform. The session is needed to read in tables as data frames, and to write data frames back to tables. In PySpark, by convention, the `SparkSession` is named `spark`, and available globally. For consistency across platforms, we always pass it into the `model` function as an explicit argument named `session`.
+- returns: a data frame (Snowpark, PySpark, or Pandas)
```

Comment:
instead of "- returns" that makesit part of the parameters list I'd write "<newline> and returns ..."